src/ess/sans/types.py

```diff
@@ -11,53 +11,36 @@
 
 import sciline
 import scipp as sc
+from ess.reduce.nexus import generic_types as reduce_gt
+from ess.reduce.nexus import types as reduce_t
 from ess.reduce.uncertainty import UncertaintyBroadcastMode as _UncertaintyBroadcastMode
 
-UncertaintyBroadcastMode = _UncertaintyBroadcastMode
-
-# 1  TypeVars used to parametrize the generic parts of the workflow
-
-# 1.1  Run types
-BackgroundRun = NewType('BackgroundRun', int)
-"""Background run: the run with only the solvent which the sample is placed in."""
-EmptyBeamRun = NewType('EmptyBeamRun', int)
-"""Run (sometimes called 'direct run') where the sample holder was empty.
-It is used for reading the data from the transmission monitor."""
-SampleRun = NewType('SampleRun', int)
-"""Sample run: the run with the sample placed in the solvent inside the sample holder.
-"""
-
-ScatteringRunType = TypeVar(
-    'ScatteringRunType',
-    SampleRun,
-    BackgroundRun,
-)
-
+BackgroundRun = reduce_gt.BackgroundRun
+CalibratedDetector = reduce_gt.CalibratedDetector
+CalibratedMonitor = reduce_gt.CalibratedMonitor
+DetectorData = reduce_gt.DetectorData
+DetectorPositionOffset = reduce_gt.DetectorPositionOffset
+EmptyBeamRun = reduce_gt.EmptyBeamRun
+Filename = reduce_gt.Filename
+Incident = reduce_gt.Incident
+MonitorData = reduce_gt.MonitorData
+MonitorPositionOffset = reduce_gt.MonitorPositionOffset
+MonitorType = reduce_gt.MonitorType
+NeXusMonitorName = reduce_gt.NeXusMonitorName
+NeXusDetector = reduce_gt.NeXusDetector
+NeXusMonitor = reduce_gt.NeXusMonitor
+RunType = reduce_gt.RunType
+SampleRun = reduce_gt.SampleRun
+ScatteringRunType = reduce_gt.ScatteringRunType
+Transmission = reduce_gt.Transmission
+TransmissionRun = reduce_gt.TransmissionRun
+SamplePosition = reduce_gt.SamplePosition
+SourcePosition = reduce_gt.SourcePosition
+
+DetectorBankSizes = reduce_t.DetectorBankSizes
+NeXusDetectorName = reduce_t.NeXusDetectorName
```

Comment:
I don't like this, but I think otherwise we would need to rewrite a lot more imports and type hints? This way users can still import from `ess.sans.types` and don't need to know which ones are defined in `ess.reduce.nexus`.... even though that is how the graph viz now displays them :(

Reply:
I know I am probably alone on this, but see this as another reason for considering #95 again.

Reply:
Has the situation changed now that we have workflow objects with default parameters set?
I guess the arguments about inspecting intermediate results have not really changed...

Reply:
You mean the default params we introduced for the widgets? That is not merged and probably still in flux, so I cannot tell yet.